Makefile.in

```diff
@@ -388,9 +387,16 @@ endif
 	@echo "Missing xsltproc.  "$@" not (re)built."
 endif
 
-$(objroot)doc/%.3 : $(objroot)doc/%.xml $(srcroot)doc/stylesheet.xsl $(objroot)doc/manpages.xsl
+$(objroot)doc/%$(install_suffix).3 : $(objroot)doc/%.xml $(srcroot)doc/stylesheet.xsl $(objroot)doc/manpages.xsl
 ifneq ($(XSLROOT),)
 	$(XSLTPROC) -o $@ $(objroot)doc/manpages.xsl $<
+# The -o option (output filename) of xsltproc may not work (it uses the
+# <refname> in the .xml file).  Manually add the suffix if so.
+  ifneq ($(install_suffix),)
+    ifneq ($(wildcard $(objroot)doc/jemalloc.3),)
+	@mv doc/jemalloc.3 doc/jemalloc$(install_suffix).3
```

Comment:
Is `$(objroot)` needed here, i.e. `@mv $(objroot)doc/jemalloc.3 $(objroot)doc/jemalloc$(install_suffix).3`?

Reply:
Yes good catch; just updated. I also noticed that the `ifneq ($(wildcard` check is not reliable (might be because the .3 file was just generated from this step). Switched to a shell `if` and now it's working.